@app/config/src/index.ts

```diff
@@ -1,13 +1,121 @@
+import { defaultNumber, minLength, parseInteger, required } from "./validate";
+
 // @ts-ignore
 const packageJson = require("../../../package.json");
 
 // TODO: customise this with your own settings!
 
-export const fromEmail =
-  '"PostGraphile Starter" <no-reply@examples.graphile.org>';
-export const awsRegion = "us-east-1";
-export const projectName = packageJson.name.replace(/[-_]/g, " ");
-export const companyName = projectName; // For copyright ownership
-export const emailLegalText =
+export const fromEmail: string = `"PostGraphile Starter" <no-reply@examples.graphile.org>`;
+export const awsRegion: string = "us-east-1";
+export const projectName: string = packageJson.name.replace(/[-_]/g, " ");
+export const companyName: string = projectName; // For copyright ownership
+export const emailLegalText: string =
```

Comment:
I'm surprised the linter does not complain about unnecessary type definitions here (since they can be implied by the values).

Reply:
I'd usually go for inferred but I wanted to ensure that people changing the validators didn't accidentally do something like change the exported type from `string` to `string | undefined` without making it explicit.